scarlet/initialization.py

```diff
@@ -11,6 +11,7 @@
 
 logger = logging.getLogger("scarlet.initialisation")
 
+
 def get_best_fit_seds(morphs, images):
```

Comment:
No, `frame` really doesn't make sense here.

Reply:
Look at https://github.com/pmelchior/scarlet/blob/factored/scarlet/initialization.py#L362-L363 . I'm basically just saying that L362 should be moved into this function so that other code has access to the projection.

Reply:
what for?

Reply:
Because it saves a line of code anywhere else we need this functionality and it doesn't cost us anything.

Reply:
I think the signature is clear this way. We use this method exactly twice, once with a box and once without. I think it can stand as is.

Reply:
Fine. I still disagree, but I don't think either of us is going to come around to the other persons point of view so we might as well resolve it.